### ISSUE TITLE
Default certificate and Debian Buster

### ISSUE THREAD

**Issue by WouldYouKindly** [SpectoLabs/hoverfly#898]
Hi.

Trying to connect to Hoverfly with curl on Debian Buster leads to 

```
curl: (60) SSL certificate problem: EE certificate key too weak
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```
even if `--cacert cert.pem` is passed.

Steps:
1. `docker create network test`.
2. `docker run --rm  -p 8888:8888 -p 8500:8500  --network test spectolabs/hoverfly:latest`
3. `docker run --rm -it --network test python:3.7.6-slim bash` (python-slim is based on debian-buster).
4. (inside container) `apt update && apt install curl wget`.
5. (inside container) `wget https://raw.githubusercontent.com/SpectoLabs/hoverfly/master/core/cert.pem > cert.pem`
6. (inside container) `export HTTPS_PROXY='{hoverfly_container_ip}:8500'`
7. (inside container) `curl https://google.com --cacert cert.pem`.

Presumably it is connected with https://wiki.debian.org/ContinuousIntegration/TriagingTips/openssl-1.1.1

Same thing happen when using `requests` python library (it uses openssl-1.1.1 under the hood).

I tried generating my own certificate and passing it to hoverfly and curl, but the same error occurred. I'm no expert at the whole SSL thing though.

**Comment by tommysitu** [comment]
It suggests that the key hoverfly using to sign the cert is too weak, could be related to this: https://github.com/SpectoLabs/hoverfly/issues/872?

**Comment by WouldYouKindly** [comment]
I sincerely have no idea. The page on debian wiki says that error `is caused by the SECLEVEL 2 setting the security level to 112 bit. This means that RSA and DHE keys need to be at least 2048 bit long. SHA-1 is no longer supported for signatures in certificates and you need at least SHA-256.`

It seems that hoverfly's cert meets these requirements?

**Comment by ns3777k** [comment]
There's indeed an issue with key generating. I hope I'll make a PR tomorrow or this week.

**Comment by tommysitu** [comment]
I think @ns3777k's PR should fix this problem

**Comment by tommysitu** [comment]
Hi @WouldYouKindly are you able to build master and try out the fix?

**Comment by tommysitu** [comment]
Forgot you use Docker. You can use the latest image for master: 
`docker pull spectolabs/hoverfly:master`

**Comment by WouldYouKindly** [comment]
@tommysitu yup, works like a charm. Thank you both!

**Comment by tommysitu** [comment]
🎉well done @ns3777k

**Comment by tommysitu** [comment]
fix released in v1.1.5
